rules-tests/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector/Fixture/add_default_nullable.php.inc

```diff
@@ -20,7 +20,7 @@ namespace Rector\Tests\Php80\Rector\Class_\ClassPropertyAssignToConstructorPromo
 
 final class AddDefaultNullable
 {
-    public function __construct(public ?string $name = null)
+    public function __construct(public $name = null)
```

Comment:
it seems regression, the `$name` property no longer typed on this case. I will look into it.

Reply:
https://github.com/rectorphp/rector-src/pull/4091